core/modules/layout/layout.admin.inc

```diff
@@ -667,18 +743,21 @@ function layout_settings_form_validate(&$form, &$form_state) {
 }
 
 /**
- * Submit handler for layout_settings_form() that updates available contexts.
+ * Submit handler for layout_settings_form() that updates available contexts
+ * and information about path matching.
```

Comment:
Function summary description should fit on a single 80 character line. You could put this onto it's own separate line:

```suggestion
 * Submit handler for layout_settings_form().
 *
 * Updates available contexts and information about path matching.
```